driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -1375,6 +1375,32 @@ static __always_inline void apply_dynamic_snaplen(struct pt_regs *regs, uint16_t
 		BPF_CORE_READ_INTO(&port_remote, sk, __sk_common.skc_dport);
 		port_local = ntohs(port_local);
 		port_remote = ntohs(port_remote);
+
+		if(port_remote == 0 && sockaddr != NULL)
+		{
+			BPF_CORE_READ_USER_INTO(&socket_family, (struct sockaddr*)sockaddr, sa_family);
```

Comment:
we should already have the `socket_family` why do we need to extract it again?

Reply:
🤦